CHANGES.rst

```diff
@@ -462,6 +462,10 @@ New Features
 
   - Improve DAOphot reader to read multi-aperture files [#3535, #4207]
 
+  - if the keys in the formats dictionary not to match the names in the names
+    dictionary then it's raises warning to show which keys are not to match in
+    keys in names Dictionary. [#4508]
```

Comment:
Thanks for adding a changelog entry.  However, it's in the wrong section--you've placed it in an existing release instead of one of the upcoming ones.  In any case I'm not sure yet what release this should be fixed in (I suppose a patch release would be fine probably though).